.github/workflows/ci.yml

```diff
@@ -24,7 +25,26 @@ jobs:
           pytest .
       - name: Test McStasScript
         run: |
-          python3 -m pip install McStasScript
+          git clone https://github.com/PaNOSC-ViNYL/McStasScript.git
+          cd McStasScript
+          pip install -e .
+          cd mcstasscript/tests
+          
+          pytest .
+          cd
+      - name: Test SimEx-Lite
+        run: |
+          git clone --recursive https://github.com/PaNOSC-ViNYL/SimEx-Lite.git
+          cd SimEx-Lite
+          pip install -e .
+          git clone https://github.com/PaNOSC-ViNYL/SimEx-Lite-testFiles testFiles
+
+          git clone -b simex https://github.com/JunCEEE/pysingfel.git
+          cd pysingfel
+          pip install ./
+          cd ..
+          
+          pytest tests
```

Comment:
Nice that projects that depend on libpyvinyl are tested when libpyvinyl is changed.